doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+
+`Contract API <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ is a logical component within the Plutus Contract package, providing an effective system for describing smart contracts that interact with wallets, DApps and the blockchain. 
+It provides the Contract API interface for writing the off-chain part of a Plutus application that is to be interpreted by an emulator or by Plutus application backend (PAB). 
+
+5. Plutus chain index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus chain index                                                           |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-chain-index-core``                       |
+|              | ``plutus-apps/plutus-chain-index``                            |
++--------------+---------------------------------------------------------------+
+
+   COMMENT: Ziyang suggests that we remove this one from the page. Does anyone have other opinions? 
+
+`Plutus chain index <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-chain-index-core>`_ 
+is an application for indexing data from the Cardano blockchain that is 
+used in the Contract Monad. The main design is for indexing information related to the 
+UTxO set in order to keep the amount of indexed information proportional to the 
+UTxO set. 
+
+6. PAB (Plutus application backend)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| PAB                                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-pab``                                    |
++--------------+---------------------------------------------------------------+
+
+`PAB <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_ 
+is a web server library for managing the state of Plutus contract instances. 
+The PAB executes the off-chain component of Plutus applications. It manages 
+application requests to the wallet backend, the Cardano node and the chain-index. 
+PAB stores the application state and offers an HTTP REST API for managing application 
+instances. 
+
+PAB wraps the contracts built with plutus-contract. It is the central point of 
+contact, integrating many Cardano components. 
+
+7. PAB (Plutus application backend) simulator 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| PAB simulator                                                                |
```

Comment:
Since we're not sure about that part given the comments below, I'd propose removing it.

Reply:
@koslambrou @asutherlandus @raduom I'll remove PAB simulator for now. We can always put it back in if you determine that it should be listed here.